rclpy/rclpy/rosnode.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2016 Erle Robotics, LLC
+#
+# A relevant part of the code has been written taking inpiration
+# from ROS 1 ros_comm package attributed to Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+import time
+import os
+import errno
+import sys
+
+from optparse import OptionParser
+
+NAME='rosnode'
+
+# TODO implement
+def _rosnode_cmd_ping(argv):
+    print("NOT IMPLEMENTED\n")
+    sys.exit(0)
```

Comment:
There is a standard Python exception called `NotImplementedError`: https://docs.python.org/3.5/library/exceptions.html#NotImplementedError